portfolio/src/main/webapp/script.js

```diff
@@ -37,12 +23,30 @@ function addRandomFact() {
  * Adds messages to the page.
  */
 function getMessages() {
-  fetch('/messages').then(response => response.json()).then((messages) => {
-    const messagesPElement = document.getElementById('messages-container');
-    messagesPElement.innerHTML = '';
-    for (i = 0; i < messages.length; i+=2) {
-      const emoji = String.fromCodePoint(parseInt(messages[i+1]));
-      messagesPElement.appendChild(createPElement(messages[i] + " " + emoji));
+  const messagesElement = document.getElementById('messages-container');
+  const buttonElement = document.getElementById('view-button');
+
+  // Toggle hide and show
+  if (messagesElement.style.display === "none") {
```

Comment:
Instead of modifying the styles directly in your JS, I would consider defining a CSS class `hidden` and then adding and removing that class to toggle an element's visibility.

This is probably one of those subjective things that comes down to personal preference, so feel free to stick with inline styles if they make more sense to you.